packages/import/src/importIssues.ts

```diff
@@ -364,3 +364,21 @@ const buildComments = async (
   }
   return `${description}\n\n---\n\n${newComments.join("\n\n")}`;
 };
+
+const createIssueWithRetries = async (
+  client: LinearClient,
+  input: Parameters<LinearClient["createIssue"]>[0],
+  retries = 3
+): ReturnType<LinearClient["createIssue"]> => {
+  try {
+    return await client.createIssue(input);
+  } catch (error) {
+    if (retries <= 0) {
```

Comment:
So, right now we don't even know if it's a rate limiting error?

Reply:
Oops, I forgot to commit that part. Done!